examples/hello_world.rs

```diff
@@ -1,9 +1,13 @@
 #[macro_use] extern crate nickel;
 
-use nickel::{Nickel, HttpRouter};
+use nickel::{Nickel, HttpRouter, Request, Response, MiddlewareResult};
+
+fn hello_world<'mw>(_req: &mut Request, res: Response<'mw>) -> MiddlewareResult<'mw> {
```

Comment:
This currently doesn't work because `MiddlewareResult` also expects a type like `()` to make it work.